frontend/src/react/components/appPermissions.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+class AppPermissions extends React.Component {
+    render() {
+        return <div className="pure-u-4">
+            <h2>{this.props.app_name}</h2>
+            <h4><em>Written by {this.props.creator}</em></h4>
+            <hr/>
+            <h3>Permissions</h3>
+            <h4>Standard Permissions</h4>
+            <em>
+                All UCL API apps that use this system are provided with this data.
+            </em>
+            <ul>
+                <li>Your Name ({this.props.user_full_name})</li>
+                <li>Your Email Address ({this.props.user_email})</li>
+                <li>Your Department ({this.props.user_department})</li>
+                <li>Your UPI ({this.props.user_upi})</li>
+            </ul>
+
+            <h4>Personal Data Permissions</h4>
+             {
+                (this.props.scopes.length == 0) &&
+                <em>
+                    This app has not requested access to any of your personal UCL data. It will only be able to see public data, such as the public timetable and room bookings information. If you are expecting this app to show you any personal data, please contact the vendor.

```

Comment:
Strike this `If you are expecting this app to show you any personal data, please contact the vendor.` imo

Adds too much information at this stage